### PR TITLE
Fix some text on Portuguese Brazilian translation.

### PR DESCRIPTION
**Pull request by Ashura-X**

### CODE REVIEW

**Comment by Ashura-X** [comment]
nt

File: language/Portuguese_Brazil/strings.po [hunk 1]
```diff
@@ -20,871 +20,871 @@ msgstr ""
...
 
 #: src/emu/ui/barcode.cpp:72
 msgid "New Barcode:"
-msgstr ""
+msgstr "Novo c�digo de barras"
 
 #: src/emu/ui/barcode.cpp:76
 msgid "Enter Code"
-msgstr ""
+msgstr "Entrar com o c�digo"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: Inserir Código

File: language/Portuguese_Brazil/strings.po [hunk 2]
```diff
@@ -20,871 +20,871 @@ msgstr ""
 
 #: src/emu/ui/auditmenu.cpp:111
 msgid "Audit in progress..."
-msgstr ""
+msgstr "Auditoria em progresso"
 
 #: src/emu/ui/barcode.cpp:72
 msgid "New Barcode:"
-msgstr ""
+msgstr "Novo c�digo de barras"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: you missed the ":" in the end of the string. Please try to keep the same punctiation and case-style in translations.

File: language/Portuguese_Brazil/strings.po [hunk 3]
```diff
@@ -20,871 +20,871 @@ msgstr ""
...
 
 #: src/emu/ui/barcode.cpp:76
 msgid "Enter Code"
-msgstr ""
+msgstr "Entrar com o c�digo"
 
 #: src/emu/ui/barcode.cpp:118
 msgid "Barcode length invalid!"
-msgstr ""
+msgstr "Tamanho do c�digo de barras inv�lido"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: again, here punctuation is not the same. You're forgetting the exclamation mark "!" in the end.

File: language/Portuguese_Brazil/strings.po [hunk 4]
```diff
@@ -20,871 +20,871 @@ msgstr ""
...
 
 #: src/emu/ui/cheatopt.cpp:143
 msgid "Reload All"
-msgstr ""
+msgstr "Recarregar Tudo"
 
 #: src/emu/ui/cheatopt.cpp:259
 msgid "Autofire Status"
-msgstr ""
+msgstr "Situa��o do Autodisparo"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: The work "Status" is already highly well understood ammong technically literate Brazilians (and portuguese speakers in general). I would keep it as "Status do Autodisparo"

File: language/Portuguese_Brazil/strings.po [hunk 5]
```diff
@@ -20,871 +20,871 @@ msgstr ""
...
 #: src/emu/ui/custmenu.cpp:195 src/emu/ui/custmenu.cpp:507
 msgid "Add filter"
-msgstr ""
+msgstr "Adicionar filtro"
 
 #: src/emu/ui/custmenu.cpp:210 src/emu/ui/custmenu.cpp:230
 #: src/emu/ui/custmenu.cpp:523 src/emu/ui/custmenu.cpp:543
 msgid "Select custom filters:"
-msgstr ""
+msgstr "Selecionar filtros customiz�veis:"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: maybe here a better term would be "Selecionar filtros personalizados:" or even "Selecionar filtros personalizáveis:"

File: language/Portuguese_Brazil/strings.po [hunk 6]
```diff
@@ -20,871 +20,871 @@ msgstr ""
...
 
 #: src/emu/ui/custui.cpp:564
 msgid "UI Colors Settings"
-msgstr ""
+msgstr "Configura��es de cores do UI"
 
 #: src/emu/ui/custui.cpp:592 src/emu/ui/selector.cpp:181
 msgid "Double click or press "
-msgstr ""
+msgstr "Duplo-clique ou pressione"
```
Comment (felipesanches) [inline language/Portuguese_Brazil/strings.po]: Apparently you need to leave the extra space here in the end of the string.

**Comment by cuavas** [comment]
Note that advice to keep the same punctuation and capitalisation may be applicable for some Portuguese, it isn't applicable for all languages.  For example in German capitalisation can affect the meaning, and Japanese uses punctuation differently.

**Comment by Ashura-X** [comment]
OK fixed all tipos like felipesanches told me !

**Pull request closed without merging**